packages/engine/Source/Scene/Cesium3DTilesVoxelProvider.js

```diff
@@ -385,6 +532,7 @@ async function getSubtreePromise(provider, subtreeCoord) {
   // Check one more time if the subtree is in the cache.
   // This could happen if there are two in-flight tile requests from the same
   // subtree and one finishes before the other.
+  // TODO: only allow one request per subtree
```

Comment:
Should this TODO be resolved here? Or should we document this in an issue?

Reply:
https://github.com/CesiumGS/cesium/issues/12457